libs/images/node-base/README.md

```diff
@@ -0,0 +1,22 @@
+# Sample Node base image
+
+Comes with Typescript so build tasks can happen in the container without referencing typescript as a dependency
+
+versions:
+  - node: 12.x
+  - typescript: 3.7.5
+
+
+USAGE:
+---
+
+
+```bash
+docker build . -t amidostacks/node
+docker tag 9175cc947ecb amidostacks/node:0.0.3
+docker tag 7a80ca09ba27 amidostacks/node:latest
```

Comment:
Two tags?

Reply:
Get latest only?

Reply:
well it's in a README but you should always tag with a version and with latest - so that is by design